packages/manager/apps/dedicated/client/app/network-security/scrubbing-center/scrubbing-center.controller.js

```diff
@@ -25,9 +27,35 @@ export default class ScrubbingCenterController {
     if (!ip) {
       this.getAllEvents();
     }
+    this.isLoading = false;
+  }
+
+  getEventsList(cursor, after, subnets, pageSize) {
+    this.isLoading = true;
+    const params = {
+      after,
+      subnets,
+    };
+    return this.networkSecurityService
+      .getEventsList({
+        cursor,
+        params,
+        pageSize,
+      })
+      .then((response) => {
+        if (response.data) {
+          this.events = this.events.concat(response.data);
+        }
+        if (response.cursor.next) {
+          this.getEventsList(response.cursor.next, after, subnets, pageSize);
+        }
```

Comment:
Hello @Steffy29 , don't forget to remove that part as soon as APIV2 next cursor will be fixed

Reply:
I don't implement your way to display pagination so I need to recall this method to get all events available for the params I set. I'm not sure that I would use you pagination way because I need the number of page I am on and the total number of events which it's not possible with your way.

Reply:
Hello @Steffy29, for now, APIV2 don't permit to manage page numbers and total results so it's weird you have to display that by using APIV2. All apps using APIV2 manage only a simple pagination with prev/next (no more information are displayed)
Moreover, it might cause performance issues as soon as data is quite big.

Reply:
Ok, let's do that but don't forget to inform API team for the next cursor issue